### ISSUE TITLE
Log In error banner doesn't announce itself to screen readers

### ISSUE THREAD

**Issue by hbillings** [18F/cms-hitech-apd#1678]
If a user enters incorrect login info, the alert banner at the top of the screen does not get announced to the user, so there is no indication of an incorrect login attempt.

### This task is done when...
 - [ ] Alert banner is read on login fail
 - [ ] Probably needs a `role=alert` and/or `aria-live=assertive`

**Comment by NAretakis** [comment]
duplicate of #1303, closing issue.
